### PR TITLE
Chat: add missing parts that are untested in dark theme and compact size mode

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

**Review by Zedwag — DISMISSED** [review]

File: e2e/testcafe-devextreme/tests/chat/etalons/Messagelist loadindicator position on initial loading (material-blue-light).png
Comment (ksercs) [inline e2e/testcafe-devextreme/tests/chat/etalons/Messagelist loadindicator position on initial loading (material-blue-light).png:1]: something went wrong here

File: e2e/testcafe-devextreme/tests/chat/typingIndicator.ts [hunk 1]
```diff
@@ -14,13 +14,29 @@ fixture.disablePageReloads`ChatTypingIndicator`
...
+  });
+
+  const darkTheme = getFullThemeName().replace('light', 'dark');
+  await testScreenshot(t, takeScreenshot, 'Typing indicator with emptyview.png', {
+    element: '#container',
+    theme: darkTheme,
+    themeChanged: async () => {
+      await chat.repaint();
+    },
+  });
```
Comment (ksercs) [inline e2e/testcafe-devextreme/tests/chat/typingIndicator.ts:34]: I see u added a screenshot taking for compact and dark + removed previous `testScheenshot`.
Do we test non-compact light now?

File: e2e/testcafe-devextreme/tests/chat/etalons/Messagelist loadindicator position on initial loading (material-blue-light).png
Comment (marker-dao) [inline e2e/testcafe-devextreme/tests/chat/etalons/Messagelist loadindicator position on initial loading (material-blue-light).png:1]: Looks strange) I think it's some problem with switching the theme in runtime. Maybe we should make a separate test?

File: e2e/testcafe-devextreme/tests/chat/typingIndicator.ts (hunk 1 above)
Comment (EugeniyKiyashko) [reply e2e/testcafe-devextreme/tests/chat/typingIndicator.ts:34]: Yes, we test both modes - default & compact when `testScreenshot` is using with `shouldTestInCompact: true` flag

File: e2e/testcafe-devextreme/tests/chat/etalons/Messagelist loadindicator position on initial loading (material-blue-light).png
Comment (EugeniyKiyashko) [reply e2e/testcafe-devextreme/tests/chat/etalons/Messagelist loadindicator position on initial loading (material-blue-light).png:1]: Etalon was updated

**Review by ksercs — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]

**Pull request merged**